browse/templates/stats/base.html

```diff
@@ -0,0 +1,14 @@
+{%- extends "base.html" -%}
+{# Do not show login status on stats pages for now, per classic #}
+
+{% block head %}
+  {{ super() -}}
+  {# TODO: upgrade d3 to v5.x? #}
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.17/d3.min.js" type="text/javascript"></script>
```

Comment:
This enables third parties surveillance capabilities of our uses.

Reply:
I've moved this to our CDN.